dont prepend ./ to the binary being lauched, it should be in $PATH

Index: Pdf4QtLaunchPad/launchdialog.cpp
--- Pdf4QtLaunchPad/launchdialog.cpp.orig
+++ Pdf4QtLaunchPad/launchdialog.cpp
@@ -120,7 +120,7 @@ void LaunchDialog::startProgram(const QString& program
     }
     else
     {
-        internalToolPath = QString("./%1").arg(program);
+        internalToolPath = QString("%1").arg(program);
     }
 
     qint64 pid = 0;
